packages/clerk-js/src/core/auth/cookies/session.ts

```diff
@@ -35,8 +36,11 @@ export const createSessionCookie = (cookieSuffix: string): SessionCookieHandler
     sessionCookie.set(token, { expires, sameSite, secure });
   };
 
+  const get = () => suffixedSessionCookie.get() || suffixedSessionCookie.get();
```

Comment:
Removing the duplication + referencing the function:

```suggestion
  const get = suffixedSessionCookie.get;
```

Reply:
nice catch probably i wanted to grab the suffixed and unsuffixed versions.